include/onnxruntime/core/common/status.h

```diff
@@ -121,16 +121,18 @@ class [[nodiscard]] Status {
 
   Status(StatusCategory category, int code);
 
-  GSL_SUPPRESS(r.11)
-  Status(const Status& other)
-      : state_((other.state_ == nullptr) ? nullptr : new State(*other.state_)) {}
-  GSL_SUPPRESS(r.11)
+  // GSL_SUPPRESS(r.11)
+  Status(const Status& other) {
+    state_ = nullptr;
+    if (other.state_ != nullptr) state_ = std::make_unique<State>(*other.state_);
+  }
+  // GSL_SUPPRESS(r.11)
   Status& operator=(const Status& other) {
     if (state_ != other.state_) {
       if (other.state_ == nullptr) {
         state_.reset();
       } else {
-        state_.reset(new State(*other.state_));
+        state_.reset(std::make_unique<State>(*other.state_).release());
```

Comment:
You can suppress the warning with "#pragma warning(disable: xxxx)". You don't have to fix the warning.